src/pages/publication/publication_page/PublicationPageAuthors.tsx

```diff
@@ -1,24 +1,70 @@
 import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Link, CircularProgress } from '@material-ui/core';
+
 import { Contributor } from '../../../types/contributor.types';
 import NormalText from '../../../components/NormalText';
-import styled from 'styled-components';
-import { Link } from '@material-ui/core';
+import useFetchUnitHierarchy from '../../../utils/hooks/useFetchUnitHierarchy';
+import { getCommaSeparatedUnitString, getDistinctContributorUnits } from '../../../utils/institutions-helpers';
 
 const StyledAuthor = styled.span`
   margin-right: 1rem;
 `;
 
+const StyledAffiliationsContainer = styled.div`
+  margin-top: 0.5rem;
+  padding-left: 1rem;
+`;
+
 interface PublicationPageProps {
   contributors: Contributor[];
 }
 
-const PublicationPageAuthors: FC<PublicationPageProps> = ({ contributors }) => (
-  <NormalText>
-    {contributors.map((contributor, index) => {
-      const { arpId, name } = contributor.identity;
-      return <StyledAuthor key={index}>{arpId ? <Link href={`/profile/${arpId}`}>{name}</Link> : name}</StyledAuthor>;
-    })}
-  </NormalText>
-);
+const PublicationPageAuthors: FC<PublicationPageProps> = ({ contributors }) => {
+  const distinctUnits = getDistinctContributorUnits(contributors);
+
+  return (
+    <>
+      <NormalText>
+        {contributors.map((contributor, index) => {
+          const { arpId, name } = contributor.identity;
+          const affiliationIndexes = contributor.affiliations?.map(
+            (affiliation) => distinctUnits.indexOf(affiliation.id) + 1
+          );
+
+          return (
+            <StyledAuthor key={index}>
+              {arpId ? <Link href={`/profile/${arpId}`}>{name}</Link> : name}
+              {affiliationIndexes?.length > 0 && <sup>{affiliationIndexes.join(',')}</sup>}
```

Comment:
hva er `<sup>`? 😅

Reply:
den lærte jeg i dag faktisk! "sup" for "superscript".
x^2  =  x<sup>2</sup>

Reply:
haha, det gikk jo bedre enn det skulle gå:
`x^2 = x<sup>2</sup>`

Reply:
<sub> for subscript

Reply:
👍